cfgov/cfgov/settings/base.py

```diff
@@ -488,7 +489,8 @@
     "'self'",
     "'unsafe-inline'",
     'fast.fonts.net',
-    'api.mapbox.com')
+    'api.mapbox.com',
+    'tagmanager.google.com')
```

Comment:
Trivial: You could add a comma at the end of this one so that next time we add one, we don't have to modify two lines :)

Reply:
I was just following the existing code.